cpp/oneapi/dal/algo/knn/backend/model_impl.hpp

```diff
@@ -18,35 +18,179 @@
 
 #include "oneapi/dal/algo/knn/common.hpp"
 #include "oneapi/dal/algo/knn/backend/model_interop.hpp"
+#include "oneapi/dal/backend/serialization.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+
+#ifdef ONEDAL_DATA_PARALLEL
+#include <src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi.h>
+#else
+#include <daal/src/algorithms/k_nearest_neighbors/bf_knn_classification_predict_kernel.h>
+#endif
```

Comment:
Why this is added to the model declaration, this should be included only at kernel level

Reply:
because of this
![image](https://user-images.githubusercontent.com/54797533/124918379-a2718280-dffd-11eb-8308-6590ee30ec80.png)

Reply:
new file was added